microcosm_pubsub/consumer.py

```diff
@@ -72,7 +73,11 @@ def parse_content(cls, consumer, message):
         Parse and validate the message.
 
         """
-        base_message = consumer.pubsub_message_codecs["_"].decode(message)
+        try:
```

Comment:
As I mentioned in [https://github.com/globality-corp/oscar/pull/158](), I think we should move the initial parsing logic into a separate construct so it can be overridden more simply.